### ISSUE TITLE
SPFX WebPart shows error in IE when deployed on a Site as WebPart

### ISSUE THREAD

**Issue by Crash753** [SharePoint/sp-dev-docs#786]
- [X] Question
- [ ] Typo
- [ ] Bug
- [ ] Additional article idea

When I test the WebPart within the workbench with IE11, everything works as it should. But when I deploy the WebPart on a Page, then it shows the following error:

Something went wrong

If the problem persists, contact the site administrator and give them the information in Technical Details.

Technical Details

[SPLoaderError.loadComponentError]:
***Failed to load component "..." (...).
Original error: ***Failed to load entry point from component "..." (...).
Original error: Error loading https://component-id.invalid/...
	Object doesn't support property or method 'startsWith'

***INNERERROR:
***Failed to load entry point from component "..." (...).
Original error: Error loading https://component-id.invalid/...
	Object doesn't support property or method 'startsWith'

The WebPart runs in the Workbench since I added the required polyfills. But on the Sharepoint Page the polyfills seem to be ignored. In Chrome everything works everywhere as expected. So I guess it must be a polyfill problem. 

Does anyone had the same problem?

I use SPFX with React and within one component I use the react-wysiwyg Rich Text Editor. (I guess the RTE- component is the culprit since when I added it, the errors shows up)

Any suggestions and help is very appreciated

Thank you

**Comment by Crash753** [comment]
Update: 
The error with startsWith only occurs when the page is in Read Mode. 
If I change to Edit Mode, the WebPart loads without any problems.
But when I save the changes and refresh the page, then the error shows up again.

SharePoint mystery...

**Comment by manishgarg1** [comment]
Hi @Crash753 , are you bundling react-wysiwyg RTE with the web part code or are you externalizing it ?

**Comment by Crash753** [comment]
Dang it... wrong button^^

**Comment by nickpape-msft** [comment]
@Crash753 , which polyfills did you add to the page? Also, are the polyfills also bundled, or are they externalized?

**Comment by colindekker** [comment]
You need to add the starts-with polyfill from for instance coreJS.
Run `npm i core-js`, then in the webpart code import the `starts-with` polyfill `corejs/fn/string/starts-with`

**Comment by Crash753** [comment]
@nickpape-msft I tried to bundle the polyfills but that does not work really well.

But I can't belief that the solution was that simple...
My own polyfills alone do not work but when I imported the polyfill 'startsWith' from coreJS as @colindekker recommended it works. But only in comination with my own polyfills.
Import all polyfills from coreJS does not work either.

Bottom line is...it works now...

Thank you all
